### DISCUSSION TITLE
Is this repo for Android?

### DISCUSSION THREAD

**Discussion by Sternbach-Software** [MicrosoftEdge/WebView2Feedback#3922] [Q&A]
I personally am a die-hard Edge fan! Great job team! Recently, the Android app got an update that made the app perform at light-speed! I want to know if I can integrate that kind of networking in my own app, and just to understand what made it so fast. If this is the right repo - great. If not, kindly point me in the right direction.

**Comment by champnic** [comment]
Hey @shmueldabomb441 - happy to meet a die-hard fan!! Most of our networking is derived from the Chromium code, and usually when we make changes there we commit to the Chromium repo. I'll find out if our mobile app is built on the regular Android WebView or something else and get back to you :)
